sale_order_line_input/views/sale_order_line_view.xml

```diff
@@ -0,0 +1,208 @@
+                <field name="qty_invoiced" sum="Invoiced" optional="show" />
+                <field name="qty_to_invoice" sum="To Invoice" optional="show" />
+                <field
+                    name="product_uom"
+                    string="UoM"
+                    attrs="{'readonly': [('state', 'in', ('sale','done', 'cancel'))]}"
+                    context="{'company_id': company_id}"
+                    groups="uom.group_uom"
+                    options='{"no_open": True}'
+                    optional="show"
+                />
+                <field
+                    name="analytic_tag_ids"
+                    attrs="{'readonly': [('state', 'in', ('done', 'cancel'))]}"
+                    groups="analytic.group_analytic_accounting"
+                    widget="many2many_tags"
+                    options="{'color_field': 'color'}"
+                    optional="show"
+                />
+                <field
+                    name="price_unit"
+                    attrs="{'readonly': ['|', ('qty_invoiced', '&gt;', 0), ('state', 'in', ('done', 'cancel'))]}"
+                    optional="show"
+                />
+                <field
+                    name="tax_id"
+                    widget="many2many_tags"
+                    options="{'no_create': True}"
+                    domain="[('type_tax_use','=','sale'),('company_id','=',company_id)]"
+                    invisible="context.get('tax_hide', False)"
+                    attrs="{'readonly': ['|', ('qty_invoiced', '&gt;', 0), ('state', 'in', ('done', 'cancel'))]}"
+                    optional="show"
+                />
+                <field
+                    name="discount"
+                    string="Disc.%"
+                    attrs="{'readonly': [('state', 'in', ('done', 'cancel'))]}"
+                    groups="product.group_discount_per_so_line"
+                    optional="show"
+                />
+                <field
+                    name="price_subtotal"
+                    widget="monetary"
+                    groups="account.group_show_line_subtotals_tax_excluded"
+                    optional="show"
+                />
+                <field
+                    name="price_total"
+                    widget="monetary"
+                    groups="account.group_show_line_subtotals_tax_included"
+                    optional="show"
+                />
+                <field name="product_updatable" invisible="1" />
+                <field name="qty_delivered_method" invisible="1" />
+                <field name="state" invisible="1" />
+                <field name="invoice_status" invisible="1" />
+                <field name="customer_lead" invisible="1" />
+                <field name="currency_id" invisible="1" />
+                <field name="price_tax" invisible="1" />
+                <field name="pricelist_id" invisible="1" />
+                <field name="company_id" invisible="1" />
+            </tree>
```

Comment:
Can you compare existing view in v14 so that the same fields with the same attributes are available?
https://github.com/odoo/odoo/blob/14.0/addons/sale/views/sale_views.xml#L444-L578

At least these changes have been made:
https://github.com/odoo/odoo/commit/b657463d56cc2bd1ec993233cf40eb384fb35700#diff-f1a4d6f572cc46e7405de6ea3b075eeaa4a46da5a4e0d4d2ebedf2db82b6ce95
https://github.com/odoo/odoo/commit/db0de87ead2333b24fcb7c69458fc27d4d058e26#diff-f1a4d6f572cc46e7405de6ea3b075eeaa4a46da5a4e0d4d2ebedf2db82b6ce95
https://github.com/odoo/odoo/commit/696ea173d3f6cb1230444c34d6f19bc4519fda04#diff-f1a4d6f572cc46e7405de6ea3b075eeaa4a46da5a4e0d4d2ebedf2db82b6ce95R570

As I mentioned in the previous migration, the idea of this module is to have the same options available as when adding a line from the sales order form view:
https://github.com/OCA/sale-workflow/pull/1502#discussion_r589517795
https://github.com/OCA/sale-workflow/pull/1502#issuecomment-829969805